packs/feats/derring-do.json

```diff
@@ -24,7 +24,78 @@
             "remaster": false,
             "title": "Pathfinder Advanced Player's Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "keep": "higher",
+                "key": "RollTwice",
+                "predicate": [
+                    "self:effect:panache",
+                    "action:tumble-through"
+                ],
+                "selector": "acrobatics-check"
+            },
```

Comment:
Instead of doing this with a bunch of REs we should condense this to a single rule with a larger predicate.

```json
"predicate": [
    "self:effect:panache",
    {
       "or": [
            "action:tumble-through",
            { "and": ["feature:battledancer", "action:perform"] }
           ...
        ]
    }
]```

Reply:
Actually, we have a different idea that will be a much larger project but end up being much cleaner.